AdblockPlusSafari/AdblockPlus.m

```diff
@@ -26,6 +26,8 @@
 static NSString *AdblockPlusDownloadedVersion = @"AdblockPlusDownloadedVersion";
 static NSString *AdblockPlusWhitelistedWebsites = @"AdblockPlusWhitelistedWebsites";
 
+static NSString *AdblockPlusSafariExtension = @"AdblockPlusSafariExtension";
```

Comment:
I wanted to separate this variable, because it has different semantic then values above (it is value and not key used in dictionary).